test/e2e_node/services/kubelet.go

```diff
@@ -264,7 +264,7 @@ func (e *E2EServices) startKubelet() (*server, error) {
 		"--kubeconfig", kubeconfigPath,
 		"--root-dir", KubeletRootDirectory,
 		"--v", LOG_VERBOSITY_LEVEL, "--logtostderr",
-		"--allow-privileged", "true",
+		"--allow-privileged=true",
```

Comment:
we should also make sure boolean flags are handled properly in `addKubeletConfigFlags`, below
https://github.com/kubernetes/kubernetes/blob/5fa4f2f6ec7669c7f9ee6e290bc91501e8fd965b/test/e2e_node/services/kubelet.go#L352-L359

Reply:
Thank you for pointing this out. Will fix by this patch and push again:
```
--- a/test/e2e_node/services/kubelet.go
+++ b/test/e2e_node/services/kubelet.go
@@ -354,7 +354,7 @@ func addKubeletConfigFlags(cmdArgs *[]string, kc *kubeletconfig.KubeletConfigura
        fs := pflag.NewFlagSet("kubelet", pflag.ExitOnError)
        options.AddKubeletConfigFlags(fs, kc)
        for _, name := range flags {
-               *cmdArgs = append(*cmdArgs, "--"+name, fs.Lookup(name).Value.String())
+               *cmdArgs = append(*cmdArgs, fmt.Sprintf("--%s=%s", name, fs.Lookup(name).Value.String()))
        }
 }
```